### PR TITLE
Add rubygem-foreman_probing package

### PR DESCRIPTION
**Pull request by dgoetz**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.19
* [ ] 1.18
* [ ] 1.17

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---
Requires #2777

### CODE REVIEW

File: comps/comps-foreman-rhel7.xml [hunk 1]
```diff
@@ -306,18 +306,14 @@
...
       <packagereq type="default">nodejs-file-loader-doc</packagereq>
       <packagereq type="default">nodejs-ipaddr.js-doc</packagereq>
       <packagereq type="default">nodejs-jquery.cookie-doc</packagereq>
       <packagereq type="default">nodejs-jquery-doc</packagereq>
       <packagereq type="default">nodejs-jquery-flot-doc</packagereq>
-      <packagereq type="default">nodejs-jquery-ujs-doc</packagereq>
-      <packagereq type="default">nodejs-jstz-doc</packagereq>
       <packagereq type="default">nodejs-multiselect-doc</packagereq>
       <packagereq type="default">nodejs-select2-doc</packagereq>
-      <packagereq type="default">nodejs-url-loader-doc</packagereq>
```
Comment (ehelms) [inline comps/comps-foreman-rhel7.xml]: This nodejs comps changes don't seem related, @dgoetz can you comment on these or drop them from the diff to merge?

Comment (ekohl) [reply comps/comps-foreman-rhel7.xml]: I think they're already in rpm/develop so a trivial rebase should be sufficient.

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
@dgoetz could you please rebase now that #2772 is merged?

**Comment by dgoetz** [comment]
@evgeni: Rebased and checks have passed.

**Comment by evgeni** [comment]
```
% koji add-pkg foreman-plugins-nightly-rhel7 tfm-rubygem-foreman_probing --owner=kojiadm
Adding 1 packages to tag foreman-plugins-nightly-rhel7
```

**Comment by evgeni** [comment]
build: http://koji.katello.org/koji/taskinfo?taskID=175472
